elements/lisk-dpos/src/dpos.ts

```diff
@@ -208,4 +208,34 @@ export class Dpos {
 			delegateListRoundOffset,
 		});
 	}
+
+	// tslint:disable-next-line: prefer-function-over-method
+	public async isDPoSProtocolCompliant(
```

Comment:
also, can you add this usage of this function in the block_procoessor_v2 where BFTProtocolCompliant is called?

Reply:
u mean a comment?

Reply:
I mean in this PR, block_processor_v2 should use this to check the dpos compliance and update/validate the reward